### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by satcomjimmy**

### CODE REVIEW

**Comment by mgrdcm** [comment]
Good catch!  Probably don't even need to keep around the old style anymore, since nothing older than Django 3 is supported.

Maybe ditch this part altogether: https://github.com/bennylope/django-organizations/pull/275/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR169-R180

And just keep the updated syntax block: https://github.com/bennylope/django-organizations/pull/275/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR184-R194

**Comment by satcomjimmy** [comment]
Noob level: I had no idea my PR was going to go to the parent repo. I thought that was just to update my fork, however, I'm glad to drop the v1 syntax and update the request.

> Good catch! Probably don't even need to keep around the old style anymore, since nothing older than Django 3 is supported.
> 
> Maybe ditch this part altogether: https://github.com/bennylope/django-organizations/pull/275/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR169-R180
> 
> And just keep the updated syntax block: https://github.com/bennylope/django-organizations/pull/275/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR184-R194

**Review by bennylope — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -172,8 +172,8 @@ main application URL conf as well as your chosen invitation backend URLs:
```
[the remark this answers is not in the document]
Comment (mgrdcm) [reply README.rst:172]: What do you think @bennylope?

[the remark this answers is not in the document]
Comment (satcomjimmy) [reply README.rst:172]: I likely have some more edits to suggest in the future, so I can add it back on a future PR

**Pull request merged**